lib/std/json/stringify.zig

```diff
@@ -33,6 +33,9 @@ pub const StringifyOptions = struct {
 
     /// Should unicode characters be escaped in strings?
     escape_unicode: bool = false,
+
+    /// When true, renders numbers outside the range `+-1<<53` (the precise integer range of f64) as JSON strings in base 10.
+    emit_big_numbers_quoted: bool = false,
```

Comment:
I suggested `emit_nonportable_numbers_as_strings` since is ambiguous whether it is referring to large values or large absolute values.

Reply:
good point. i like `emit_nonportable_numbers_as_strings` also. :+1:

Reply:
Ok, renamed to `emit_nonportable_numbers_as_strings`.

While on this topic, it may be unnecessary to specify the term `emit` since this is apart of StringifyOptions, however, omitting this now would make it inconsistent with the current set of options.  I'll leave it up to someone else to decide if we should make a change.